examples/40_advanced/example_metrics.py

```diff
@@ -46,6 +46,12 @@ def error_wk(solution, prediction, extra_argument):
     return np.mean(solution != prediction)
 
 
+def metric_which_needs_x(solution, prediction, x_data, extra_argument):
+    # custom function defining accuracy
+    assert x_data is not None
+    return np.mean(solution[extra_argument, :] == prediction[extra_argument, :])
```

Comment:
Can we illustrate with some arbitrary use of `x_data` for the example, i.e. make rows with NaNs count for twice as much?

Reply:
I updated the example with a more meaningful metric